### PR TITLE
Metadata

### PR DESCRIPTION
**Pull request by gatemezing**
Replaced rdfs label by skos preflabels in other languages

### CODE REVIEW

**Review by pchampin — APPROVED** [review]
LGTM. I'm a little uncomfortable with the 'modified' date, which may get outdated if we don't update them manually... but I understand that there is some value in putting them there as well. We could possibly add script in the Makefile to update them automatically, but that should not defer the merging of this PR ;) https://xkcd.com/1319/

File: tests/manifest.jsonld [hunk 1]
```diff
@@ -2,7 +2,20 @@
     "@context": "manifest-context.jsonld",
     "@id" : "manifest",
     "@type": "Manifest",
-    "label": "RDF-star test suite",
+    "label": {"en": "RDF-star test suite"},
+    "prefLabel": {"fr": "Les tests de RDF-star",
+              "es": "Conjunto de pruebas de RDF-star"
+            },
```
[the remark this answers is not in the document]
Comment (gkellogg) [reply tests/manifest.jsonld:8]: Sorry, I don't see that make-jsonld-manifests.rb has been modified in this PR.

File: tests/trig/eval/manifest.ttl [hunk 2]
```diff
@@ -9,10 +9,20 @@ PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#>
...
-    rdfs:label "TriG-star Evaluation Tests" ;
-    mf:entries
+   rdfs:label "TriG-star Evaluation Tests"@en ;
+   rdfs:label "La suite des tests pour évaluation de TriG-star"@fr;
+   rdfs:label "Conjunto de pruebas para evaluar Trig-star"@es;
+   dct:issued "2021-06-21"^^xsd:date ;
+   rdfs:seeAlso <https://w3c.github.io/rdf-tests/> ;
+   dct:modified "2021-07-18"^^xsd:date ;
+   dct:licence <https://www.w3.org/Consortium/Legal/2008/03-bsd-license> ;
+   dct:creator [ foaf:homepage <https://w3c.github.io/rdf-star/> ; foaf:name " W3C RDF-star Working Group" ] ;
```
[the remark this answers is not in the document]
Comment (gkellogg) [reply tests/trig/eval/manifest.ttl]: I don't see an entry for this in the manifest.jsonld version of the file, which should be created when the make-jsonld-manifests script is run.

**Review by gkellogg — CHANGES_REQUESTED** [review]
As mentioned in another comment, I don't see any changes to the make-jsonld-manifests.rb script to automate the update of the JSON-LD versions of the manifests. Running this script creates these files, and also ensures that the Turtle and JSON-LD versions of the files are isomorphic.

**Comment by pchampin** [comment]
@gkellogg I missed that and you are right... all the more that the JSONLD and HTML versions of the trig/syntax and trig.eval manifest contain errors.

**Comment by pchampin** [comment]
to be fair, the errors predate @gatemezing's PR, so I think we can merge the PR and fix this problem afterwards

**Comment by gkellogg** [comment]
Very well, I'll take on the action to update the scripts once this is merged.

**Review by gkellogg — APPROVED** [review]

**Comment by gkellogg** [comment]
I have changes staged for making both HTML and JSON-LD manifests once this PR is merged. Neither were building TriG manifests, but I've also made other changes to the HTML manifests.

**Pull request merged**